DateInput/README.md

```diff
@@ -0,0 +1,63 @@
+
+# DateInput Interface
+
+**Status**: Proposed
+
+| Name        | ID                                                                |
+| :---------- | :---------------------------------------------------------------- |
+| DateInput | e1229b97060a95ebb8024fafbb181acbbae2264669c2b43928b350bf830ae038  |
+
+
+## Description
+
+Interface for date input.
+
+## Functions
+
+`select` - prints prompt message to the user and returns dates entered by user.
+
+arguments:
+
+	answerId: uint32
+    prompt: bytes
+    min: uint32
+    max: uint32
+    exclude: uint32[]
```

Comment:
If you exclude the date, you exclude the range or an array of ranges, not an array of values.